migrations/sqlite/20240321211121_block.up.sql

```diff
@@ -0,0 +1,7 @@
+-- Add up migration script here
+
+CREATE TABLE IF NOT EXISTS "block" (
```

Comment:
I like writing them as idempotent generally but it would be important to know if the table was wrong. 

I guess I could either remove the IF, or run a SELECT columna,..columnz from table; as a verify/assert that it worked.. if the columns don't match it should fail. sqitch offers a hook for verify but it's not quite the same in sqlx.

Reply:
I went with the SELECT to verify approach.. made sure they do fail with an invalid column

```
---- events::tests::test_validate_time_event stdout ----
thread 'events::tests::test_validate_time_event' panicked at one/src/events.rs:434:58:
called `Result::unwrap()` on an `Err` value: while executing migrations: error returned from database: (code: 1) no such column: tx_hash1```